server.js

```diff
@@ -1,33 +1,99 @@
 import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
+import goldenGlobesData from './data/golden-globes.json'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
 const port = process.env.PORT || 8080
 const app = express()
 
 // Add middlewares to enable cors and json body parsing
+
 app.use(cors())
 app.use(bodyParser.json())
 
+
 // Start defining your routes here
+
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Hi! For all the data: http://localhost:8080/nominations, filter by category, http://localhost:8080/nominations?category=Best%20Original%20Score%20-%20Motion%20Picture, filter by year of the nomination: http://localhost:8080/nominations?year=2012, only the winner for a specific year: http://localhost:8080/nominations?year=2012&win=true, ')
```

Comment:
You can make it nicer by creating the index.html file and send it to the user, or use templating engine like ejs: https://ejs.co/